crafting.lua

```diff
@@ -51,14 +51,14 @@ minetest.register_craft({
 })
 
 minetest.register_craft({
-	output = "digiline_routing:diode",
+	output = "digiline_routing:diode 2",
```

Comment:
Please don’t, that makes connector count inconsistent.

Reply:
Github has suggestions:
```suggestion
	output = "digiline_routing:diode",
```

Reply:
That wasn’t that good idea, this comment was for all 3 recipes...